docs/anatomy/anatomy-github-apps.md

```diff
@@ -0,0 +1,73 @@
+---
+id: github-apps
+title: GitHub Apps
+---
+
+
+The GitHub Apps feature enables integration of repositories in GitHub with KintoHub using third party OAuth.
+
+## Connecting GitHub
+ 
+1. Login to **KintoHub**
+2. Click on the **Create Service** button at the top right corner
+3. Choose a service from the list of services provided
+4. Click on the **Connect Github** button
+
+![Connect Github btn](/img/anatomy/connect-github-btn.png)
+:::note
+If your GitHub account has multiple organisations linked to it, you will be provided with an option to choose the appropriate organisation and access the repositories under that organisation on KintoHub.
+:::
+5. To select only preferred repositories, enable the option **Only select repositories** and then make your selection
+6. Click on he **Install & Autorize** button
```

Comment:
action - Autorize -> Authorize? Make sure you have a spell checker.

Reply:
changed Autorize -> Authorize

Reply:
Changed Autorize -> Authorize.